microsoft-azure-sb/3.0/modules/ROOT/pages/azure-service-bus-connector-examples.adoc

```diff
@@ -0,0 +1,361 @@
+			<ee:message>
+				<ee:set-payload><![CDATA[%dw 2.0
+output application/json
+---
+{
+	key1: "value1",
+	key2: "value2 for Queue"
+}]]></ee:set-payload>
+			</ee:message>
+			<ee:variables>
+				<ee:set-variable variableName="messageAttributes"><![CDATA[%dw 2.0
+output application/json
+---
+{
+	messageId: "messageId1234",
+	correlationStrategy: "AUTO",
+	correlationId: "correlationId",
+	label: "my_label",
+	partitionKey: "the_partition_key",
+	timeToLive: 60,
+	ttlTimeUnit: "SECONDS",
+	scheduledEnqueueTimeUtc: "2018-12-09 03:01:00.000",
+	properties: {
+		property1: "value1",
+		property2: "value2"
+	}
+}]]></ee:set-variable>
+			</ee:variables>
+		</ee:transform>
+		<azure-service-bus-messaging:send doc:name="Queue Send" doc:id="50b3592c-f777-4667-b042-00fd266120e8" config-ref="Azure_Service_Bus_Messaging_Connector_Config" destinationName="${queue.name}" messageId="#[vars.messageAttributes.messageId]" correlationId="#[vars.messageAttributes.correlationId]" label="#[vars.messageAttributes.label]" sendCorrelationStrategy="#[vars.messageAttributes.correlationStrategy]" partitionKey="#[vars.messageAttributes.partitionKey]" timeToLive="#[vars.messageAttributes.timeToLive]" scheduledEnqueueTimeUtc='#[vars.messageAttributes.scheduledEnqueueTimeUtc as LocalDateTime {format: "yyyy-MM-dd HH:mm:ss.SSS"}]' timeToLiveTimeUnit="#[vars.messageAttributes.ttlTimeUnit]" properties="#[vars.messageAttributes.properties]">
+		</azure-service-bus-messaging:send>
+	</flow>
+	<flow name="queue-receiver-flow" doc:id="9b39aaa5-e81f-4257-b65e-d4f48c8ef2d8" >
+		<http:listener doc:name="Receive From Queue" doc:id="386337a0-99c9-45cf-a81f-95d5e06f0b74" config-ref="HTTP_Listener_config" path="/receive"/>
+		<azure-service-bus-messaging:receive doc:name="Receive From Queue" doc:id="05c8c754-79f6-4873-8329-af7f48f01d3b" config-ref="Azure_Service_Bus_Messaging_Connector_Config" destinationName="${queue.name}"/>
+		<ee:transform doc:name="Attributes to Json" doc:id="a509a361-4bd7-4c6e-81cc-39ad36e7b625">
+			<ee:message>
+			</ee:message>
+			<ee:variables >
+				<ee:set-variable variableName="jsonAttributes" ><![CDATA[%dw 2.0
+output application/json
+---
+attributes]]></ee:set-variable>
+			</ee:variables>
+		</ee:transform>
+		<logger level="INFO" doc:name="Log Attributes" doc:id="94e7254c-a197-4fa2-b437-b64719acb23d" message="#[vars.jsonAttributes]" />
+		<logger level="INFO" doc:name="Log the message body" doc:id="d802b081-f3fc-4ec2-a23c-96bc0db3863c" message="#[payload]" />
+	</flow>
+</mule>
+----
++
+. When you are prompted to regenerate `doc:id` values, click *Yes*.
+. Save the project.
+. Right-click in the Studio canvas and select *Run project <project-name>* to run the project.  
+
+== Example Batch Send and Receive
+
+This application flow uses the *Send batch* and *Receive Batch* operations to send and receive batches of messages to and from an Azure Service Bus queue. 
```

Comment:
I think per Kim, we are supposed to use backticks for operations unless we are talking about the GUI but this is not a clear case. We should discuss when we have time so we can be in synch.

Reply:
Yeah, this is UI. I don't put them in bacticks unless they are references to code. If this were referring to code, it would be `azure-service-bus-messaging:send` All of the operations have a friendly name for the UI that's diff from the code.